src/types.nim

```diff
@@ -206,9 +208,16 @@ type
     gif*: Option[Gif]
     video*: Option[Video]
     photos*: seq[string]
+    communityNote*: Option[CommunityNote]
 
   Tweets* = seq[Tweet]
 
+  CommunityNote* = object
+    title*: string
+    subtitle*: string
+    footer*: string
+    url*: string
```

Comment:
It seems to me that the only important thing which changes is `subtitle`, the rest can just be hardcoded, turning this into just `communityNote*: Option[string]`